content/docs/searching.md

```diff
@@ -312,17 +312,17 @@ All dialog-free search actions do preserve the current search options set in the
 
-If you've highlighted some groups of text in this manner, and you wish to copy those sections, the Copy Styled Text submenu of the Search menu will allow you to do that.  Quick search for previously marked text is possible via the Jump Up or Jump Down submenu choices.
+If you've highlighted some groups of text in this manner, and you wish to copy those sections, the **Copy Styled Text** submenu of the Search menu will allow you to do that.  Quick search for previously marked text is possible via the **Jump Up** or **Jump Down** submenu choices.
 
-Note: Unfortunately, the Mark All submenu name can cause some confusion between an identically-named action button in the Mark tab of the Find family dialog.  The two types of "marking" are different but do share some features.  For example, the Copy Styled Text submenu commands will allow you to copy text that has been styled with number 1 through 5 styling OR text that has been marked using the Mark tab of Find.
+Note: In older versions of Notepad++, the **Mark All** submenu name can cause some confusion between an identically-named action button in the Mark tab of the Find family dialog.  The two types of "marking" are different but do share some features.  For example, the Copy Styled Text submenu commands will allow you to copy text that has been styled with number 1 through 5 styling OR text that has been marked using the Mark tab of Find.  This has been improved by renamnig the menu to **Style All Ocurrences of Token**.
 
-You can also cause all occurrences of the word at the caret to get dynamically highlighted if you activate Smart Highlighting; the mark style then is Settings -&gt; Style Configurator -&gt; Global Styles , Smart Highlighting. You may choose there whether the matching should be sensitive to case.
+You can also cause all occurrences of the word at the caret to get dynamically highlighted if you activate Smart Highlighting; the mark style then is [Settings &gt; Style Configurator &gt; Global Styles &gt; Smart Highlighting](../preferences/#global-styles). You may choose there whether the matching should be sensitive to case.
 
-You activate smart highlighting through **Settings > Preferences > MISC > Smart highlighting, Enable smart highlighting**. By default, the highlighting is case insensitive, which may be a problem sometimes. Then just toggle **Settings &gt; Preferences &gt; MISC &gt; Highlighting is case sensitive** on.  (See also [Style Configurator](../preferences/#style-configurator) and [Preferences](../preferences/#preferences).)
+You activate smart highlighting through [Settings > Preferences > Highlighting > Smart highlighting > Enable](../preferences/#highlighting). By default, the highlighting is case insensitive; if this is a problem, just toggle [Settings &gt; Preferences &gt; Highlighting &gt; Smart Highlighting > Matching: Case Sensitive](../preferences/#highlighting) on.
```

Comment:
If you mention "case" here, should also mention "whole word"?  Fine with me if this is deemed "too heavy".

Reply:
reworded